vital/tests/test_sfm_constraints.cxx

```diff
@@ -0,0 +1,297 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+#include <arrows/proj/geo_conv.h>
```

Comment:
I'm not sure if this is strictly necessary, but from what I've seen all the existing test files have something like this at the top
```suggestion
// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.

/// \file
/// \brief [insert description]

#include <arrows/proj/geo_conv.h>
```